example/ViewPagerAndroidExample.android.js

```diff
@@ -100,8 +100,17 @@ export default class ViewPagerAndroidExample extends React.Component {
       position: 0,
       offset: 0,
     },
+    pages: []
   };
 
+  componentDidMount() {
+    const pages = [];
+    for (let i = 0; i < PAGES; i++) {
+      pages.push(this.createPage(i));
+    }
+    this.setState({ pages });
```

Comment:
It's better not to call `setState` in `componentDidMount` as it will trigger another re-render. We can do it in the constructor like:

```js
constructor(props) {
  super(props);
  this.state = {
     // Whatever you need
  }
}
```